core/os/os.h

```diff
@@ -306,6 +306,7 @@ class OS {
 		CURSOR_VSPLIT,
 		CURSOR_HSPLIT,
 		CURSOR_HELP,
+		CURSOR_CUSTOM,
```

Comment:
Instead of adding `CURSOR_CUSTOM` I'd just add a `p_shape` parameter to `set_custom_mouse_cursor()`.
That way, a game that makes heavy use of UI nodes can easily override any of the shapes.

Reply:
You're right, it makes more sense.